lib/ember_cli/runner.rb

```diff
@@ -32,9 +35,11 @@ def run!(command)
 
     private
 
-    def write(output, streams:)
-      streams.each do |stream|
-        stream.write(output)
+    def redirect(stream, pipes:)
+      Tee.open(*pipes, stdout: nil) do |pipe|
+        while line = stream.gets
```

Comment:
In capybara-webkit, we used [IO.copy_stream](http://ruby-doc.org/core-2.3.0/IO.html#method-c-copy_stream) in a background thread.

https://github.com/thoughtbot/capybara-webkit/blob/master/lib/capybara/webkit/connection.rb#L103